README.md

```diff
@@ -121,11 +121,33 @@ Check the [authentication example](examples/authentication/authentication.go) fo
 
 ### Key flow
 
-To use the key flow, you need to have a service account key and an RSA key-pair. To configure it, follow this steps:
+To use the key flow, you need to have a service account key and an RSA key-pair.
+To configure it, follow this steps:
+
+    The following instructions assume that you have created a service account and assigned it the necessary permissions, e.g. project.owner.
 
 1. In the Portal, go to `Service Account -> Service Account Keys` and create a key.
    - You can create your own RSA key-pair or have the Portal generate one for you.
-2. Save the content of the service account key and the corresponding private key by copying them or saving them in a file.
+2. Save the content of the service account key and the corresponding private key by copying them or saving them in a file. The expected format of the service account key is the following:
+   ```json
+   {
+     "id": "uuid",
+     "publicKey": "public key",
+     "createdAt": "2023-08-24T14:15:22Z",
+     "validUntil": "2023-08-24T14:15:22Z",
+     "keyType": "USER_MANAGED",
+     "keyOrigin": "USER_PROVIDED",
+     "keyAlgorithm": "RSA_2048",
+     "active": true,
+     "credentials": {
+       "kid": "string",
+       "iss": "my-sa@sa.stackit.cloud",
+       "sub": "uuid",
+       "aud": "string",
+       "privateKey": "private key when generated by the SA service"

```

Comment:
I'd write this one as optional, because the Portal doesn't give this if you generate the key pair on your own